src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(String color, int radius) {
+        super.setColor(color);
+        this.radius = radius;
+    }
+
+    public Circle() {
```

Comment:
do we need a constructor without parameters? fix in all figures classes please.

Reply:
In the new solution I will need empty constructors with no parameters, since you said to declare new objects in class fields and not in methods